docs/ceph-csi-upgrade.md

```diff
@@ -114,7 +114,8 @@ git checkout v3.7.2
 ```
 
 ```console
-Warning: kubectl apply should be used on resource created by either kubectl create --save-config or kubectl apply
+Warning: kubectl apply should be used on resource created by
```

Comment:
The `console` output examples are expected to be on a single line. These lines may become long, and that is OK in these cases. Is there no way to mark these kind of blocks so that they can contain long lines? Maybe in the configuration file, or by adding an annotation?